packages/rancher-logging/overlay/templates/loggings/eks/logging.yaml

```diff
@@ -20,6 +20,12 @@ spec:
     tolerations:
       {{ toYaml . | nindent 6 }}
     {{ end }}
+      - key: "cattle.io/os"
```

Comment:
If `Values.fluentbit_tolerations` is empty, does the `tolerations` node still exist above for this to fall under ?

Reply:
Good catch. It was actually the issue I ran into since I assumed we always specify a tolerance in ```values.yaml```. Fixed!